rita_common/src/tunnel_manager/mod.rs

```diff
@@ -48,9 +46,8 @@ use std::time::{Duration, Instant};
 struct NetworkMonitorQueue {}
 
 lazy_static! {
-    /// Partial storage for tunnel manager data, Tunnel Manager is not fully migrated to async/await so this stores
-    /// data that other modules need to send to tunnel manager for it to dequeue later.
-    static ref TUNNEL_MANAGER: Arc<RwLock<NetworkMonitorQueue>> = Arc::new(RwLock::new(NetworkMonitorQueue::default()));
+    pub static ref TUNNEL_MANAGER: Arc<RwLock<TunnelManager>> =
```

Comment:
does this need to be public? ideally we would have getters and setters in this file, mostly to make it easier to avoid deadlocks